staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -113,6 +114,8 @@ type Config struct {
 	// FallThroughHandler is the final HTTP handler in the chain.  If it is nil, one will be created for you.
 	// It comes after all filters and the API handling
 	FallThroughHandler *mux.PathRecorderMux
+	// SharedInformerFactory provides shared informers for resources
+	SharedInformerFactory informers.SharedInformerFactory
```

Comment:
You need to add a poststarthook to call `.Start` on these about here: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/config.go#L407 .  Maybe a name like with something like `generic-apiserver-start-informers`